src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -21,9 +21,9 @@ export const useFormSend = () => {
     if (wallet) {
       try {
         const path = "m/44'/60'/0'/0/0" // TODO (technojak) get path and asset precision from asset-service
-        const adapter = chainAdapter.byChain(ChainIdentifier.Ethereum)
+        const adapter = chainAdapter.byChain(data.asset.chain as unknown as ChainIdentifier)
```

Comment:
Is this because unchained and assetService use different types?

Reply:
Yep. More reason to set a global types package